src/work-allocation/containers/available-tasks/available-tasks.component.ts

```diff
@@ -1,6 +1,8 @@
 import { Component } from '@angular/core';
+import { Observable } from 'rxjs';
```

Comment:
This has been added but there's nothing else in the file that suggests it's needed. Redundant?

Reply:
removing